### ISSUE TITLE
Help!

### ISSUE THREAD

**Issue by aldoglez24** [cbaez-lex/python-repuve#1]
Hey, I am working on retreiving the same data of Repuve, shall we discuss to unify efforts?

thanks!

**Comment by cbaez-lex** [comment]
yes sure, it's solved now using another technology because the pyseract isn't working properly

**Comment by aldoglez24** [comment]
Could you send me an email? aldoalejandrog@gmail.com
